static/js/settings_org.js

```diff
@@ -436,6 +443,9 @@ function update_dependent_subsettings(property_name) {
         case "realm_message_content_allowed_in_email_notifications":
             set_message_content_in_email_notifications_visiblity();
             break;
+        case "realm_enable_spectator_access":
+            set_enable_spectator_access();
+            break;
```

Comment:
I don't think either of the changes in this file are required, so I removed them.  They're both intended solely for situations where there are two settings -- things Just Work if it's an isolated setting.  So we may need something like this logic for whether to display "Who can create web-public streams": https://github.com/zulip/zulip/issues/20287.